docs/about.md

```diff
@@ -1,3 +1,11 @@
+---
+html_meta:
+  "description": "Inspiration for creating an API. Design decisions for an intuitive usage in common development tasks."
+  "property=og:description": "Inspiration for creating an API. Design decisions for an intuitive usage in common development tasks."
+  "property=og:title": "About"
+  "keywords": "inspiration, design decisions, Plone, development"
```

Comment:
For all keywords values, I would make sure that they all have at least "Plone" and "API".